src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1390,7 +1391,7 @@
                                                                        Value="Saved" />
                                                         </MultiDataTrigger.Conditions>
                                                         <Setter Property="Text"
-                                                                Value="{Binding Path=FileName, Converter={StaticResource PathToFileNameConverter}}" />
+                                                                Value="{Binding Converter={StaticResource WorkspaceToFileNameConverter}}" />
```

Comment:
here we bind to the dataContext directly by just using binding - this is like doing `Binding = Path.`